homeassistant/components/ukraine_alarm/strings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_location%]"
+    },
+    "error": {
+      "invalid_api_key": "[%key:common::config_flow::error::invalid_api_key%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]",
+          "region": "Region"
+        },
+        "description": "Set up Ukraine Alarm integration. To generate API key go to https://api.ukrainealarm.com/"
```

Comment:
Please use `description_placeholders` for text that shouldn't be translated, like URLs.

Reply:
Set up the Ukraine...

To generate an API key...

Reply:
Updated.